README.MD

```diff
@@ -216,3 +216,29 @@ There is an alert in Application Insights called `Elite2-api - Inactivity alert`
 
 If the alert fires then look for any recent releases of prison-api that may have introduced a problem. If not then check in the `nomisapi-prod` Kubernetes cluster looking for any errors - look at events, ingresses, certificates etc. Finally it could be some kind of network issue - check for problems in Azure AKS.
 
+
+### Offender images
+
+* Offender images can be set using the endpoint PUT /api/images//offender/{offenderNo}
+* This requires a token with the role ROLE_SYSTEM_USER
+* The body of the request should be a base64-encoded JPEG file
+* The optional query parameter ?fullSizeImage=true can be used if this is a full-sized image.
```

Comment:
Nomis creates both full size and thumbnail by default - so the image uploaded is full size (assumed) and then a thumbnail is generated and stored as well

Reply:
Does Nomis use standard sizes for the full and thumbnail images to get around differences in cameras used at different sites?

Reply:
I could get the original size, and if larger scale down for the fullsize stored image, and scale down again for the thumbnail?

Reply:
Implemented a scaling method, so going to find what the standard full and thumbnail image sizes are, and implement those.
`private byte[] scaleImage(int width, int height, byte[] source) throws IOException, IllegalArgumentException {
        InputStream is = new ByteArrayInputStream(source);
        BufferedImage bin = ImageIO.read(is);
        BufferedImage bout = new BufferedImage(width, height, bin.getType());
        Graphics2D g2d = bout.createGraphics();
        g2d.drawImage(bin, 0, 0, width, height, null);
        g2d.dispose();
        ByteArrayOutputStream baos = new ByteArrayOutputStream();
        ImageIO.write(bout, "jpg", baos);
        return baos.toByteArray();
    }`